site/data/roadmap.yaml

```
@@ -0,0 +1,51 @@
+- item: Continue working on issues
+  status: In progress
+  url: https://github.com/zaproxy/zaproxy/issues
+  year: 2022
+- item: Networking overhaul
+  status: Finished
+  url: https://www.zaproxy.org/blog/2022-02-10-new-zap-networking-layer/
+  year: 2022
+- item: Continue improving scan rules
+  status: In progress
+  year: 2022
+- item: Improve modern web app handling
+  status: In progress
+  year: 2022
+- item: Add permanent DB support
+  status: In progress
+  url: https://github.com/zaproxy/zap-extensions/pull/3699
+  year: 2022
+- item: Migrate Spider out of core
+  status: In progress
+  url: https://www.zaproxy.org/blog/2022-08-30-spider-move/
+  year: 2022
+- item: Migrate manual request editor out of core
+  status: In progress
+  year: 2022
+- item: Update minimum Java version to 11
+  status: In progress
+  year: 2022
+- item: Automation Framework enhancements
+  status: In progress
+  year: 2022
+- item: Release 2.12
+  status: Not started
+  year: 2022
+- item: "Networking: Initial HTTP/2 support"
+  status: Not started
+  year: 2022
+
+- item: Continue working on issues
+  status: Not started
+  url: https://github.com/zaproxy/zaproxy/issues
+  year: 2023
+- item: Networking changes in add-ons
+  status: Not started
+  year: 2023
+- item: Networking layer maintenance
+  status: Not started
+  year: 2023
+- item: Move of core functionality to add-ons
+  status: Not started
+  year: 2023
```

```suggestion
- item: Move core functionality to add-ons
  status: In Progress
  year: 2022
```
I believe we are already doing this with the spider, manual request editor, networking layer, etc. and so this could be marked "In Progress".